e2e/interchaintestv8/chainconfig/kurtosis.go

```diff
@@ -136,7 +138,7 @@ func SpinUpKurtosisPoS(ctx context.Context) (EthKurtosisChain, error) {
 
 	// Wait for the chain to finalize
 	beaconAPIClient := ethereum.NewBeaconAPIClient(beaconRPC)
-	err = testutil.WaitForCondition(10*time.Minute, 5*time.Second, func() (bool, error) {
+	err = testutil.WaitForCondition(30*time.Minute, 5*time.Second, func() (bool, error) {
```

Comment:
Why was this increased?

Reply:
It's to allow for sepolia/mainnet configs